python/py-glad2/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-glad2
+version             2.0.4
+revision            0
+homepage            https://github.com/Dav1dde/glad
+
+categories-append   devel
+license             MIT
+maintainers         {i0ntempest @i0ntempest} openmaintainer
+
+description         Multi-Language Vulkan/GL/GLES/EGL/GLX/WGL Loader-Generator based on the official specs
+long_description    ${description}
+
+checksums           rmd160  5ad534a0247b985353bb4889674f35f83ff06865 \
+                    sha256  ede1639f69f2ba08f1f498a40a707f34a609d24eb2ea0d6c9364689a798cf7d0 \
+                    size    539569
+
+python.versions     39 310 311 312
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-jinja2
```

Comment:
this doesn't look correct; `jinja` should likely be also a runtime dependency? In that case it should be `depends_lib`, additionally `py-setuptools` is automatically added because of `python.pep517 yes`, which is the default for Python 3.7 and above, so that explicit dependency can be removed here.

Reply:
Thanks, changes made.